README.md

```diff
@@ -1 +1,63 @@
-# Product.Library.DotNet
\ No newline at end of file
+
+[![NuGet Badge](https://img.shields.io/nuget/v/LightstonePlatform.Products)](https://www.nuget.org/packages/LightstonePlatform.Products/)
+# Getting Started
+
+Lightstone Platform Products is a library intended to allow for the surfacing of your products on the Lightstone platform. This package provides you with the required .Net classes to be able to respond to platform requests.
+
+## Install Lightstone Platform Products via NuGet
+
+```
+PM> Install-Package LightstonePlatform.Products -Version 0.1.5
```

Comment:
This will change as per the latest "released" version(i.e. not a prerelease version) , Not sure how we will update this when a new version is released ?

Reply:
Easy we remove the version part

Reply:
Once we get to a major version we can update this and add back version numbers for tagged releases